src/nat/front_ends/fastapi/job_store.py

```
@@ -83,101 +176,213 @@
+                        var = Variable(name=job_id, client=client)
+                        try:
+                            future = await var.get(timeout=0)
+                            if isinstance(future, Future):
+                                await client.cancel([future], force=True, reason="Expired job cleanup")
+
+                        except TimeoutError:
+                            pass
+
+                        var.delete()
+                        successfully_expired.append(job_id)
+                    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)